csrc/device_lower/validation.cpp

```diff
@@ -950,68 +950,6 @@ void validateMmaTensors(MmaOp* mma) {
       "MMA op cannot directly take double buffered register input, put a set stage before.");
 }
 
-//! Note and TODO:
-//!   Currently relying on ldmatrix to
-//!     obtain the correct data layout for turing/ampere
-//!     mma's.
-//!   This restriction will eventually not
-//!    be necessary once the scatter swizzle is ready.
-void validateTuringMmaInput(TensorView* tv) {
```

Comment:
These validation does not make sense. Although ldmatrix and mma ops are often used together, they are not married. They can be used alone without the other.